modal/app.py

```diff
@@ -586,17 +592,14 @@ def function(
         # Or, pass (request, limit) to additionally specify a hard limit in MiB.
         memory: Optional[Union[int, tuple[int, int]]] = None,
         ephemeral_disk: Optional[int] = None,  # Specify, in MiB, the ephemeral disk size for the Function.
+        min_containers: Optional[int] = None,  # Minimum number of containers to keep warm, even when Function is idle.
+        max_containers: Optional[int] = None,  # Limit on the number of containers that can be spun up for this Function
+        buffer_containers: Optional[int] = None,  # Number of additional idle containers to maintain under active load
+        scaledown_window: Optional[int] = None,  # Max amount of time a container can remain idle before scaling down.
```

Comment:
Not sure about this name, but the might have already been a discussion on that elsewhere?

"scaledown" feels like something more global imo, something that affects the whole pool of containers, which is only indirectly/partially true in this case.

Related - what happens if you adjust the scaledown window using `update_autoscaler` on a function with live containers - will the existing running containers have their scaledown_window updated or will they still respect the value from when they started up? (don't remember how this is implemented)

Reply:
Yeah we discussed extensively, it's probably my least favorite of the names here, but at this point we've made all the backend changes and I think we should value consistent frontend / backend terminology in the absence of a clear alternative.

Reply:
FWIW the primary motivation here was to avoid confusion arising from the fact that we don't always honor the full window as a "timeout"; containers aren't guaranteed to stay up for x seconds after becoming idle, rather, they are guaranteed not to be idle for more than x seconds.